data/officers.js

```diff
@@ -1,72 +1,87 @@
 export default [
 	{
-		name: 'Matthew Wang',
+		name: 'Ava Asmani',
 		title: 'President',
-		image: '/images/officers/MattWang.jpg',
+		image: '/images/officers/AvaAsmani.jpg',
 	},
 	{
-		name: 'Tina Huang',
+		name: 'Zack Pakin',
 		title: 'External Vice President',
-		image: '/images/officers/TinaHuang.jpg',
+		image: '/images/officers/ZackPakin.jpeg',
 	},
 	{
-		name: 'Evan Zhong',
+		name: 'Sujay Jain',
 		title: 'Internal Vice President',
-		image: '/images/officers/EvanZhong.jpg',
+		image: '/images/officers/SujayJain.jpeg',
 	},
 	{
-		name: 'Asha Kar',
+		name: 'Christina Tong',
 		title: 'Co-President, Hack',
-		image: '/images/officers/AshaKar.jpg',
+		image: '/images/officers/ChristinaTong.jpg',
 	},
 	{
-		name: 'Eugene Lo',
+		name: 'Nareh Agazaryan',
 		title: 'Co-President, Hack',
-		image: '/images/officers/EugeneLo.jpg',
+		image: '/images/officers/NarehAgazaryan.jpg',
 	},
 	{
-		name: 'Cindy Zhang',
-		title: 'President, ACM-W',
-		image: '/images/officers/CindyZhang.jpg',
+		name: 'Aman Oberoi',
+		title: 'President, AI',
+		image: '/images/officers/AmanOberoi.jpg',
 	},
 	{
-		name: 'Justin Yi',
-		title: 'President, AI',
-		image: '/images/officers/JustinYi.jpg',
+		name: 'Paige Brown',
+		title: 'Co-President, ACM-W',
+		image: '/images/officers/PaigeBrown.jpg',
 	},
 	{
-		name: 'Christian Loanzon',
-		title: 'Co-President, Game Studio',
-		image: '/images/officers/ChristianLoanzon.jpg',
+		name: 'Ethan Wong',
+		title: 'Co-President, ACM-W',
+		image: '/images/officers/EthanWong.jpg',
+	},
+	{
+		name: 'Robert Lee',
+		title: 'President, ICPC',
+		image: '/images/officers/RobertLee.jpg',
 	},
 	{
-		name: 'Peter Sutarjo',
+		name: 'Ray Hsiao',
 		title: 'Co-President, Game Studio',
-		image: '/images/officers/PeterSutarjo.jpg',
+		image: '/images/officers/RayHsiao.jpg',
 	},
 	{
-		name: 'Jacob Zhang',
-		title: 'President, ICPC',
-		image: '/images/officers/JacobZhang.jpeg',
+		name: 'Caroline Wang',
+		title: 'Co-President, Game Studio',
+		image: '/images/officers/CarolineWang.jpg',
 	},
 	{
-		name: 'Henry Chang',
+		name: 'Jerry Xu',
 		title: 'Co-President, Cyber',
-		image: '/images/officers/HenryChang.jpg',
+		image: '/images/officers/JerryXu.jpg',
 	},
 	{
-		name: 'Joshua Lee',
+		name: 'Stephen Kelman',
 		title: 'Co-President, Cyber',
-		image: '/images/officers/JoshLee.jpg',
+		image: '/images/officers/StephenKelman.png',
 	},
 	{
-		name: 'Sophie Schoenmeyer',
+		name: 'Milo Kearney',
 		title: 'President, Teach LA',
-		image: '/images/officers/SophieSchoenmeyer.jpg',
+		image: '/images/officers/MiloKearney.jpg',
+	},
+	{
+		name: 'Eden Yu',
+		title: 'Vice President, Teach LA',
+		image: '/images/officers/EdenYu.jpg',
+	},
```

Comment:
Typically, we don't include VPs on the leadership page - though feel free to override me on this decision! It does seem that the other committees with VPs (Design, ACM-W) swapped over to doing Co's.

Reply:
On the topic of adding people to the leadership page, I know that Teach LA has a new role split of Milo as the TLA General President and Jiin as the TLA Dev President, so I think that there's a case to be made for her to be added onto the leadership page as well!

Reply:
I decided on adding the TLA VP since none of the other committees as of now have VPs iirc, but I will reach out to Ava to confirm! 
Also think it's a good idea to add TLA Dev!